Emergency compliments failing in CI intermittently

In [this run](https://github.com/pmxbot/pmxbot/actions/runs/1118131469), observe that `test_emergency_compliment` fails on Python 3.6 but not on 3.9. In a earlier run, it failed on 3.9. It's passing for me locally. Interestingly, the error is 404:

```
self = <Response [404]>

    def raise_for_status(self):
        """Raises :class:`HTTPError`, if one occurred."""
    
        http_error_msg = ''
        if isinstance(self.reason, bytes):
            # We attempt to decode utf-8 first because some servers
            # choose to localize their reason strings. If the string
            # isn't utf-8, we fall back to iso-8859-1 for all other
            # encodings. (See PR #3538)
            try:
                reason = self.reason.decode('utf-8')
            except UnicodeDecodeError:
                reason = self.reason.decode('iso-8859-1')
        else:
            reason = self.reason
    
        if 400 <= self.status_code < 500:
            http_error_msg = u'%s Client Error: %s for url: %s' % (self.status_code, reason, self.url)
    
        elif 500 <= self.status_code < 600:
            http_error_msg = u'%s Server Error: %s for url: %s' % (self.status_code, reason, self.url)
    
        if http_error_msg:
>           raise HTTPError(http_error_msg, response=self)
E           requests.exceptions.HTTPError: 404 Client Error: Not Found for url: https://spreadsheets.google.com/feeds/list/1eEa2ra2yHBXVZ_ctH4J15tFSGEu-VTSunsrvaCAV598/od6/public/values?alt=json

.tox/python/lib/python3.6/site-packages/requests/models.py:953: HTTPError

```

So it must be an issue with spreadsheets occasionally returning not found... maybe a bug in the spreadsheet.

For now, I'll just hobble the test.

--- a/pmxbot/util.py
+++ b/pmxbot/util.py
@@ -2,6 +2,7 @@ import random
 import warnings
 import itertools
 import logging
+import operator
 
 import requests
 import bs4
@@ -99,13 +100,16 @@ def lookup_acronym(acronym, limit=3):
 
 @jaraco.functools.once
 def load_emergency_compliments():
-    compurl = (
-        'https://spreadsheets.google.com/feeds/list/'
-        '1eEa2ra2yHBXVZ_ctH4J15tFSGEu-VTSunsrvaCAV598/od6/public/values'
-        '?alt=json'
+    sheet_id = '14sYGzhjFAMLFA90wFUZ-4biW_UmwHXj5r9p2zQtGiaM'
+    sheet_name = 'sheet'
+    api_key = pmxbot.config['Google API key']
+    url = (
+        f'https://sheets.googleapis.com/v4/spreadsheets/{sheet_id}'
+        f'/values/{sheet_name}?alt=json&key={api_key}'
     )
-    doc = http.open(compurl).json()
-    return [entry['title']['$t'] for entry in doc['feed']['entry']]
+    doc = http.open(url).json()
+    breakpoint()
+    return list(map(operator.itemgetter(0), doc['values']))
 
 
 def passagg(recipient, sender):

--- a/pmxbot/util.py
+++ b/pmxbot/util.py
@@ -2,7 +2,6 @@ import random
 import warnings
 import itertools
 import logging
-import operator
 
 import requests
 import bs4
@@ -100,16 +99,9 @@ def lookup_acronym(acronym, limit=3):
 
 @jaraco.functools.once
 def load_emergency_compliments():
-    sheet_id = '14sYGzhjFAMLFA90wFUZ-4biW_UmwHXj5r9p2zQtGiaM'
-    sheet_name = 'sheet'
-    api_key = pmxbot.config['Google API key']
-    url = (
-        f'https://sheets.googleapis.com/v4/spreadsheets/{sheet_id}'
-        f'/values/{sheet_name}?alt=json&key={api_key}'
-    )
-    doc = http.open(url).json()
-    breakpoint()
-    return list(map(operator.itemgetter(0), doc['values']))
+    url = 'https://www.dropbox.com/s/2i54optwv59ykm2/compliments.txt?dl=1'
+    doc = http.open(url).text
+    return doc.splitlines()
 
 
 def passagg(recipient, sender):

--- a/tests/conftest.py
+++ b/tests/conftest.py
@@ -1,3 +1,5 @@
+import os
+
 import pytest
 import jaraco.functools
 from jaraco.context import ExceptionTrap
@@ -30,3 +32,11 @@ def needs_wordnik(needs_internet):
 
 def pytest_configure():
     pytest.check_internet = check_internet
+
+
+@pytest.fixture
+def google_api_key(monkeypatch):
+    key = os.environ.get('GOOGLE_API_KEY')
+    if not key:
+        pytest.skip("Need GOOGLE_API_KEY environment variable")
+    monkeypatch.setitem(pmxbot.config, 'Google API key', key)

--- a/tests/unit/test_commands.py
+++ b/tests/unit/test_commands.py
@@ -26,14 +26,6 @@ def onetrue(*args):
     return len(truthiness) == 1
 
 
-@pytest.fixture
-def google_api_key(monkeypatch):
-    key = os.environ.get('GOOGLE_API_KEY')
-    if not key:
-        pytest.skip("Need GOOGLE_API_KEY environment variable")
-    monkeypatch.setitem(pmxbot.config, 'Google API key', key)
-
-
 class TestCommands:
     @classmethod
     def setup_class(cls):

--- a/tests/unit/test_util.py
+++ b/tests/unit/test_util.py
@@ -9,7 +9,7 @@ def test_lookup(needs_wordnik):
 
 
 @pytest.mark.xfail(reason="#97: Google is unreliable")
-def test_emergency_compliment(needs_internet):
+def test_emergency_compliment(needs_internet, google_api_key):
     assert util.load_emergency_compliments()
 
 

--- a/tests/unit/test_util.py
+++ b/tests/unit/test_util.py
@@ -8,8 +8,7 @@ def test_lookup(needs_wordnik):
     assert util.lookup('dachshund') is not None
 
 
-@pytest.mark.xfail(reason="#97: Google is unreliable")
-def test_emergency_compliment(needs_internet, google_api_key):
+def test_emergency_compliment(needs_internet):
     assert util.load_emergency_compliments()
 
 

